APCAppCore/APCAppCore/Library/Scheduler/APCScheduleDebugPrinter.m

```diff
@@ -249,6 +264,23 @@ + (NSString *) stringFromDate: (NSDate *) date
     return result;
 }
 
+- (NSString *) stringWithMillisecondsFromDate: (NSDate *) date
```

Comment:
Another thought: add `stringWithMilliseconds` to a NSDate Category? So no need to involve printer in this case.